hack/kubernetes/allinone_aliyun.sh

```diff
@@ -96,27 +115,33 @@ EOF
 }
 
 install_kubelet_ubuntu() {
-    KUBE_URL="https://storage.googleapis.com/kubernetes-release/release/$RELEASE_UBUNTU/bin/linux/amd64"
-    wget "$KUBE_URL/kubeadm" -O /usr/bin/kubeadm
-    wget "$KUBE_URL/kubelet" -O /usr/bin/kubelet
-    wget "$KUBE_URL/kubectl" -O /usr/bin/kubectl
+    if [ -z "$RELEASE_UBUNTU" ]; then
+        echo "RELEASE_UBUNTU can't be null" >&2
+        exit 1
+    fi
+    wget -O- --read-timeout=20 --progress=bar:force:noscroll https://raw.githubusercontent.com/manlge/resources/master/kubernetes/$RELEASE_UBUNTU/kubernetes-linux-amd64.tar.xz|tar -C /usr/bin -xJ
```

Comment:
@manlge I think we should keep the url is the official. if there is GFW issue, I think our user will find a way out. 😄